runhouse/main.py

```diff
@@ -610,138 +610,140 @@ def get_clusters_from_den():
     return asyncio.run(aget_clusters_from_den())
 
 
-@cluster_app.command("list")
-def cluster_list():
-    """Load Runhouse clusters"""
-    import sky
+def _create_output_table(
+    total_clusters: int, running_clusters: int, displayed_clusters: int
+):
+    displayed_running_clusters = (
```

Comment:
wondering if we should have some sort of pagination feature here, meaning if for example there are 100 clusters and we show 10 when the command is first run, we could make it easy for the user to toggle through the complete list. wdyt? 

cc @mkandler - we're doing something similar for the resources page in Den right now also